src/pages/index.js

```diff
@@ -1,75 +1,117 @@
+import Api from "../components/Api.js";
 import "../pages/index.css";
 import Card from "../components/Card.js";
 import FormValidator from "../components/FormValidator.js";
```

Comment:
Please, delete `this.disableButton()` from line 74 in `FormValidator.js`


You need to **disable** the button only in a `then` block after a **successful response** to be able to click it again if there is a server error.